packages/analytics/src/AnalyticsProvider.tsx

```diff
@@ -73,6 +73,10 @@ export const _AnalyticsProvider: FC<IAnalyticsProvider> = ({ children, ...props
       reset: function () {
         proxyToIntegrations(integrations, 'reset', Array.from(arguments));
       },
+      revenue: function () {
+        if (!integrations.includes('amplitude')) return;
```

Comment:
this is not correct, this should make no assumptions about the integration

Reply:
amplitude is the only integration with revenue, if we try to call revenue on the other integrations it will crash, won't it? I can remove the checks and do it normally, I thought it best to restrict it to the only integration that can call it